projects/rocr-runtime/runtime/hsa-runtime/CMakeLists.txt

```diff
@@ -268,7 +268,7 @@ if (UNIX)
 set ( IMAGE_SUPPORT ${IMAGE_SUPPORT} CACHE BOOL "Build with image support (default: ON for x86, OFF elsewise)." )
 else()
   # Force IMAGE_SUPPORT to be OFF
-  set(IMAGE_SUPPORT OFF CACHE BOOL "Build with image support (forced to OFF)" FORCE)
+  set(IMAGE_SUPPORT ON CACHE BOOL "Build with image support (forced to ON)" FORCE)
```

Comment:
Could you combine this logic with the UNIX logic above? Basically remove if (UNIX) check now